.vscode/launch.json

```diff
@@ -5,24 +5,24 @@
     "version": "0.2.0",
     "configurations": [
         {
-            "name": "app",
-            "cwd": "src\\app",
+            "name": "ApplicationTemplate - Debug",
             "request": "launch",
-            "type": "dart"
-        },
-        {
-            "name": "app (profile mode)",
-            "cwd": "src\\app",
+            "type": "dart",
+            "toolArgs": ["--dart-define", "ENV=Development"]
+          },
+          {
+            "name": "ApplicationTemplate - Staging",
```

Comment:
QA

Reply:
Doing that?

Reply:
I though we wanted to do it now, but maybe it could be worth to do it later if I don't have enough time